examples/app_commands/basic.py

```diff
@@ -0,0 +1,71 @@
+from typing import Optional
+
+import discord
+from discord import app_commands
+
+MY_GUILD = discord.Object(id=0)  # replace with your guild id
+
+
+class MyClient(discord.Client):
+    def __init__(self, *, intents: discord.Intents, application_id: int):
+        super().__init__(intents=intents, application_id=application_id)
+        # A CommandTree is a special type that holds all the application command
+        # state required to make it work. This is a separate class because it
+        # allows all the extra state to be opt-in.
+        # Whenever you want to work with application commands, your tree is used
+        # to store it and work with it.
+        # Note: When using commands.Bot instead of discord.Client, the bot will
+        # maintain its own tree instead.
+        self.tree = app_commands.CommandTree(self)
+
+    # In this basic example, we just synchronize the app commands to one guild,
+    # without requiring to set the guild to every command individually by just copying them over.
+    # By doing so we don't have to wait up to an hour until they are shown to the end-user.
```

Comment:
```suggestion
    # In this basic example, we just synchronize the app commands to one guild.
    # Instead of specifying a guild to every command, we copy over our global commands instead.
    # By doing so we don't have to wait up to an hour until they are shown to the end-user.
```